### DISCUSSION TITLE
vcpkg install failure

### DISCUSSION THREAD

**Discussion by neutr1n02001** [pelicanmapping/osgearth#1853] [Support Q&A]
I'm following the instructions 
vcpkg install osgearth:x64-windows

all dependency packages are installed fine, then I got this error

-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: ninja -v
    Working Directory: D:/dev/vcpkg/buildtrees/osgearth/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      D:\dev\vcpkg\buildtrees\osgearth\config-x64-windows-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:345 (vcpkg_execute_required_process)
  ports/osgearth/portfile.cmake:34 (vcpkg_configure_cmake)
  scripts/ports.cmake:142 (include)

Error: Building package osgearth:x64-windows failed with: BUILD_FAILED

How to fix this?

**Comment by plevy** [comment]
This looks like a general vcpkg failure than an osgEarth specific problem:
Command failed: ninja -v

Can you paste in the log from here:  D:\dev\vcpkg\buildtrees\osgearth\config-x64-windows-out.log

**Comment by neutr1n02001** [comment]
I checked the log, here's the error message there.
CMake Error at CMakeModules/FindOSG.cmake:148 (find_library):
  Could not find cfitsio_LIBRARY_RELEASE using the following names: cfitsio
Call Stack (most recent call first):
  D:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake:788 (_find_package)
  CMakeLists.txt:131 (find_package)

So it's looking for this library cfitsio but not found. Anyway, I was able to build from source code in 3.2 with no issue. The code in master branch is currently broken though.

**Reply by plevy** [reply]
First, glad you were able to build successfully.

In the vcpkg ports,  it pulls a specific revision of osgEarth, not master.  There are also patches that may or may not work on later versions of osgEarth. So if you change the revision and SHA of the version specified in vcpkg to a later version, that may not work.

That error looks like cfitsio was not installed correctly and you can try vcpkg install cfitsio:x64-windows 
next time

Thank you
